app/models/order.rb

```diff
@@ -0,0 +1,11 @@
+class Order < ApplicationRecord            
+    validates :title, presence: true, length: {minimum: 5}
+    validates :description, presence:true, length: {minimum: 5}
+    enum status: %i(created accepted inProcess finished cancelled)
+
+    has_many :order_requests
```

Comment:
Aca deberia ser dependant no? Cosa de que si elimino el pedido, se eliminen todas las request asociadas a el
```suggestion
    has_many :order_requests, dependent: :destroy
```

Reply:
No es necesario por lo que he probado y visto, lo elimina solo por la asociación entre tablas al ser has_and_belongs_to_many

Reply:
Me equivoque, lo que puse arriba es el de categories_orders, para este caso puede ser si mas correcto eliminar todos los orders_request. Aunque eliminar pedidos es necesario ? tal vez es mejor dejarlos marcados con estado eliminado

Reply:
No sé si va a ser necesario eliminar pedidos, pero yo igual haría lo que dice @rafael-castelo porque refleja la realidad de esa relación y, si en un futuro se necesita, ya está hecho.